src/patternfly/utilities/Alignment/examples/alignment-simple-example.hbs

```diff
@@ -1,14 +1,14 @@
-<div class="pf-u-text-align-left">
+{{#> alignment alignment--type="left"}}
   Text left
```

Comment:
super picky but can this be "Text Left" to match the others.

Reply:
yep!